product-service/src/handlers/create-product-topic.ts

```diff
@@ -0,0 +1,47 @@
+import AWS from 'aws-sdk';
+
+const { SNS_ARN, SNS_REGION } = process.env;
+
+const publishSns = ({ sns, topicArn, messageAttributes, message, subject }) =>
+  new Promise((resolve, reject) => {
+    sns.publish({
+      TopicArn: topicArn,
+      Message: message,
+      Subject: subject,
+      MessageAttributes: messageAttributes
+    }, (error, response) => {
+      if (error) {
+        reject(error);
+      } else {
+        resolve(response);
+      }
+    })
+  });
+
+const createProductTopic = async event =>  {
+  console.log('Event data: ', event);
+
+  try {
+    const messages = event.Records[0].body;
```

Comment:
There can be more than one record

Reply:
If the batchSize=1, it will be one. But I updated based on your comments